src/main/java/org/launchcode/liftoffrecipetracker/models/Recipe.java

```diff
@@ -121,15 +138,24 @@ public void setImage(String image) {
 
 	public List<Category> getCategories() { return categories; }
 
+	public List<Beverage> getBeverages() {
+		return beverages;
+	}
+
+	public void addBeverages(List<Beverage> beverages) {
```

Comment:
Not sure how this got switched around, but it should look like the category list. 

`public void addBeverages(List<Beverage> beverages) {
		this.beverages.addAll(beverages);`